docs/development/deploying-at-scale/index.md

```diff
@@ -46,3 +46,31 @@ Thanks for flying REANA 🚀
 
 !!! note
     Note that you can deploy REANA in different namespaces by passing `--namespace` to `helm install`. Remember to pass `--create-namespace` if the namespace you want to use does not exist yet. For more information on how to work with namespaces see the [official documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+
+## Advanced deployment scenarios
+
+### High availability
+
+The services running the REANA system are critical for the platform to properly work, therefore it is a good technique to deploy them in different nodes than the ones used to run user workflows. To achieve this:
+
+**1.** Create or ask for a multinode Kubernetes cluster. For example, using the `reana-dev` tool:
+
+```console
+$ reana-dev cluster-create --worker-nodes 2
```

Comment:
Hmm, perhaps we can use abstract example so that the recipe would be ready for non-reana-dev users.  We could say here:

```
1. Create a multi-node Kubernetes cluster and check your nodes:

$ kubectl get nodes
NAME    STATUS   ROLES    AGE   VERSION
node1   Ready    master   97m   v1.18.2
node2   Ready    <none>   97m   v1.18.2
node3   Ready    <none>   97m   v1.18.2
node4   Ready    <none>   97m   v1.18.2
```

and then we label node 2 as infrastructure and node 3 as runtime in the text below?